### DISCUSSION TITLE
How to disable Bounding Box?

### DISCUSSION THREAD

**Discussion by AtsMultanen** [vladmandic/face-api#125] [Q&A] [answered]
Is there any way to disable the bounding box whilst still keeping the landmark outlines? e.g. from the original repo's example:
![image](https://user-images.githubusercontent.com/35270358/192318480-e24f8fba-7a59-4170-8378-bbea911df3c8.png)
https://justadudewhohacks.github.io/face-api.js/face_and_landmark_detection/

Ideally we'd just like to hide the score (and keep the box), however I can't see doing that without forking this library and modifying `drawDetections.ts`

**Comment by vladmandic** [comment, marked as answer]
don't use built-in helper draw functions, draw outline yourself.  
see example: <https://github.com/vladmandic/face-api/blob/dea225bbeb66c5a68f2657cbcc82ff58517d8aff/demo/webcam.js#L32>

**Reply by AtsMultanen** [reply]
Ah, that's helpful. Thanks!
